CONTRIBUTING.md

```diff
@@ -20,8 +20,15 @@ short summary of the most important parts:
 * Always use `self` for the first argument to instance methods.
 * Always use `cls` for the first argument to class methods.
 * Use one leading underscore only for non-public methods and instance variables,
-  such as `_data`.
+  such as `_data`. Do not activate name mangling with `__` unless necessary.
+* If there is a pair of `get_x` and `set_x` methods, they should instead be a
+  proper property, which is easy to do with the built-in `@property` decorator.
 * Constants should be `CAPITALIZED_SNAKE_CASE`.
+* When importing a function, try to avoid renaming it with `import as` because
+  it introduces cognitive overhead to track yet another name.
+* When deriving another module’s class (such as `unittest.TestCase`), reuse the
```

Comment:
As my explanation in overall comments, `unittest.TestCase` is a mixin actually. We can raise another case to explain this point.